docs/client.md

```diff
@@ -29,6 +29,8 @@ Set the `files` property of `eik.json` with paths to client side
 asset files in your project relative to the `eik.json` file.
 Eg. if you have a `scripts.js` file in an assets directory, the `js.input` value will be `assets/scripts.js`
 
+> 👉 Instead of specifying Eik configuration in an `eik.json` file, it is also possible to define the same values in `package.json` under an `eik` key instead.
+
```

Comment:
Do we need this here? I feel that our documentation in general explains the same concepts multiple times (we should address that in general). That is tedious for the reader and hard for us to keep up to date due to having to update multiple places when doing changes. Do you think its an ok approach to explain the config in the "overview eik json" document and not mention this here?